### PR TITLE
Add documentation for creating new fuzz test targets.

### PR DESCRIPTION
**Pull request by arik-so**

### CODE REVIEW

File: fuzz/README.md [hunk 1]
```diff
@@ -84,4 +93,38 @@ export RUSTFLAGS="--cfg=fuzzing"
...
+```shell
+mv hfuzz_workspace/fuzz_target/SIGABRT.PC.7ffff7e21ce1.STACK.[…].fuzz ./test_cases/$TARGET/
+```
+
 This will reproduce the failing fuzz input and yield a usable stack trace.
+
+
+## How do I add a new fuzz test?
+
+1. The easiest approach is to take one of the files in `fuzz/src/`, such as e. g.
```
[the remark this answers is not in the document]
Comment (arik-so) [reply fuzz/README.md]: I think just "such as" reads the nicest, so I removed "e. g."

**Review by valentinewallace — COMMENTED** [review]
Looks good mod Matt's feedback. Nice to get more fuzzing docs

File: fuzz/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dunxen) [reply fuzz/README.md]: Cool, works too!

**Review by dunxen — APPROVED** [review]

**Review by vincenzopalazzo — APPROVED** [review]

File: fuzz/README.md [hunk 2]
```diff
@@ -84,4 +84,36 @@ export RUSTFLAGS="--cfg=fuzzing"
...
+`process_network_graph.rs`, and duplicate it, renaming the new file to something more
```
[the remark this answers is not in the document]
Comment (valentinewallace) [reply fuzz/README.md:110]: Did we wanna resolve this @arik-so?

[the remark this answers is not in the document]
Comment (arik-so) [reply fuzz/README.md:110]: nope. I wanna land the graph sync PR 😛

**Pull request merged**